lib/metadata_hook.rb

```diff
@@ -4,7 +4,7 @@ def metadata
       language: {
         name: 'sqlite',
         icon: { type: 'devicon', name: 'sql' },
-        version: 'v0.2.2',
+        version: SqliteVersionHook::VERSION,
```

Comment:
This line is actually not necessary, since the *runner version* is automatically added to metadata by the mumukit on runtime - check for example the top level attribute `version` in http://sqlite.runners.mumuki.io/info. 

I'd recommend inserting here the language or technology version instead, like for example the *version of sqlite*.

For instance, in prolog runner - http://prolog.runners.mumuki.io/info - we have `swi-prolog 6.6.4`.

Reply:
:+1: